cpp/tests/io/PointCloudIO.cpp

```diff
@@ -470,7 +470,27 @@ TEST_P(ReadWritePC, UpdateProgressCallback) {
     }
 }
 
+// Test In Memory XYZ
+TEST(ReadWritePC, InMemoryXYZ) {
+    geometry::PointCloud pc_start;
+    Eigen::Vector3d one(1, 1, 1);
```

Comment:
Understood suggestion 1, a bit unclear about suggestion 2. Do you mean casting dv.data() to unsigned char without any conversions since I assume that might not read unless we convert the numbers to string first and then cast

Reply:
Added suggestion 1 in 1984a20